SCons/SConf.py

```diff
@@ -729,6 +734,9 @@ def _createDir( self, node ):
             if not os.path.isdir( dirName ):
                 os.makedirs( dirName )
 
+    def _set_conftest_node(self, node):
+        node.attributes.conftest_node = 1
```

Comment:
why introduce a setter-style method to implement a single setting? feels very java-like :(  at least mark it staticmethod, self isn't used.